packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -0,0 +1,170 @@
+   */
+  resultData: observationPropShape,
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   *  If the Result value has an appended comment.
+   */
+  isUnverified: PropTypes.bool,
+  /**
+   *  If the Result value has not been authenticated and committed to patient chart.
+   */
+  isModified: PropTypes.bool,
+  /**
+   *  If the Result value has been modified from it's original value for the same clinically documented event & datetime.
+   */
+  hasComment: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * Used by Flowsheet Result Cell to hide icons because it displays them in different positions.
+   */
+  hideAccessoryDisplays: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+  isUnverified: false,
+  isModified: false,
+  hasComment: false,
+  hasResultError: false,
+  hasResultNoData: false,
+  hideAccessoryDisplays: false,
+};
+
+const createIcons = (resultData) => (resultData.isUnverified
+  ? (<IconUnverified className={cx('icon-unverified')} />)
+  : (
+    <React.Fragment>
+      {resultData.isModified ? (<IconModified className={cx('icon-modified')} />) : null}
+      {resultData.hasComment ? (<IconComment className={cx('icon-comment')} />) : null}
+    </React.Fragment>
+  )
+);
+
+const createSecondaryDisplays = (resultData) => (
+  <React.Fragment>
+    {resultData.conceptDisplay ? (<div className={cx('concept-display')}>{resultData.conceptDisplay}</div>) : null}
+    {resultData.datetimeDisplay ? (<div className={cx('datetime-display')}>{resultData.datetimeDisplay}</div>) : null}
+  </React.Fragment>
+);
+
+const createClinicalResultDisplay = (resultData, hideUnit, isTruncated, hideAccessoryDisplays) => {
+  const isStatusInError = !isEmpty(resultData.status) ? checkIsStatusInError(resultData.status) : false;
+  const decoratedResultClassnames = cx([
+    'decorated-result-display',
+    { truncated: isTruncated },
+    { 'status-in-error': isStatusInError },
+  ]);
+  return (
+    <React.Fragment>
+      <div className={decoratedResultClassnames}>
+        <div className={cx('result-display')}>
+          <ConditionalWrapper
+            key={`del-Observation-${resultData.eventId}`}
```

Comment:
The `eventId` is an optional prop in the `observationPropShape`. If we're relying on its presence it may need to be required.  Keys in this context might not be necessary. Static children are usually safe, child arrays are not. But if we want to keep the keys we should ensure eventId is present.